suites/pre-release/http302/googleCredentialsTest.js

```diff
@@ -0,0 +1,70 @@
+Feature('Google Credentials - BDcat testing for release sign off');
+
+// To be executed with GEN3_SKIP_PROJ_SETUP=true
+// No need to set up program / retrieve access token, etc.
+const { expect } = require('chai');
+const fenceProps = require('../../../services/apis/fence/fenceProps.js');
+const { interactive, ifInteractive } = require('../../../utils/interactive.js');
+const {
+  getAccessTokenHeader,
+  requestUserInput,
+} = require('../../../utils/apiUtil');
+
+// Test elaborated for DataSTAGE but it can be reused in other projects
+const TARGET_ENVIRONMENT = process.env.GEN3_COMMONS_HOSTNAME || 'qa-dcp.planx-pla.net';
```

Comment:
where are you executing these tests? in staging or qa-dcp? 
Lets mention the staging envs instead of qa-dcp